packages/vhd-lib/src/createVhdStreamWithLength.integ.spec.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-env jest */
+
+import execa from 'execa'
+import fs from 'fs-extra'
+import rimraf from 'rimraf'
+import tmp from 'tmp'
+import { createReadStream, createWriteStream } from 'fs-promise'
+import { fromEvent, pFromCallback } from 'promise-toolbox'
+import { Slicer } from 'pipette'
```

Comment:
`pipette` is no longer maintained, could [stream-chunk-promise](https://github.com/noblesamurai/stream-chunk-promise) do the job?

Reply:
it doesn't seem to work out of the box:
`error stream-chunk-promise@1.0.2: The engine "node" is incompatible with this module. Expected version "6.x". Got "8.12.0"
error Found incompatible module
`

Reply:
I'll look for another one.

If none fits our needs we can publish a new one.

Reply:
https://github.com/emilsivervik/get-buffer?

Reply:
it seems to work for one shot reading, but not for subsequent reads. It destroys the stream after a successful reading.

Reply:
we had get-stream already in the project.